ndarray-rand/src/lib.rs

```diff
@@ -10,9 +10,9 @@
 //!
 //! See [**`RandomExt`**](trait.RandomExt.html) for usage examples.
 
-use rand::distributions::Distribution;
 use rand::rngs::SmallRng;
 use rand::{thread_rng, Rng, SeedableRng};
+use rand_distr::Distribution;
```

Comment:
I think we should be able to continue using `rand::distributions::Distribution` instead of `rand_distr::Distribution`. (`rand_distr::Distribution` is a re-export of `rand::distributions::Distribution`.)